sdk/file_sdk/UnitTests/zip_ut/zip_ut.pro

```diff
@@ -0,0 +1,30 @@
+REPO_ROOT = $$PWD/../../../..
+DESTDIR   = $$REPO_ROOT/bin/tests
+TARGET = zip_tests
+
+QT       += testlib
```

Comment:
Your tests doesn't run in the server

Reply:
yours too no? you have this one in xmp too

---
In reply to: [139380590](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139380590) [](ancestors = 139380590)

Reply:
my tests run on the server, you need to configure it. check my first pull request on how to do it

---
In reply to: [139405659](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139405659) [](ancestors = 139405659,139380590)